Automattic/woocommerce-payments
# Make iDEAL icon to look like the other payment methods

Fixes #2718 

#### Changes proposed in this Pull Request

Make iDEAL icon to look like the other payment methods

- Replaces the svg icon for iDEAL so it looks like the other payment methods.

Before

![image](https://user-images.githubusercontent.com/2612426/129203982-f41619ba-9e91-4a75-85d3-545972b62a08.png)

After

![image](https://user-images.githubusercontent.com/2612426/129204010-dc3cea52-bab6-44dd-802e-90bdcbae28b6.png)

![image](https://user-images.githubusercontent.com/2612426/129204057-ef985ad9-f91f-448d-8fd1-fa6933dd1731.png)

#### Testing instructions

- See in the payment method settings the new icon
- See in the payment detail view the new icon

-------------------
